internal/cache/controller_obj_cache.go

```diff
@@ -1517,6 +1514,20 @@ func (c *AviObjCache) AviPopulateOneVsHttpPolCache(client *clients.AviClient,
 						}
 					}
 				}
+				if rule.Match != nil {
```

Comment:
nit: This can be refactored as 
`if rule.Match == nil {
      continue
}
if rule.Match.Path == nil {
   continue
}
.
.
`
To reduce code complexity

Reply:
Combined all conditions with **and**.